ddtrace/settings/config.py

```diff
@@ -112,6 +112,9 @@ def header_is_traced(self, header_name):
         """
         return self._http.header_is_traced(header_name)
 
+    def get_service(self, default=None):
+        return self.service if self.service is not None else get_service_legacy(default=default)
```

Comment:
A comment here would be great

this is... use the globally configured service name, or else, fallback to DD_SERVICE_NAME, or else the integration default?

Reply:
The other thought I had to implement this.

In `IntegrationConfig` have a default setting for `service` for all integrations, which defaults to `DD_<integration>_SERVICE[_NAME](?)` or `None`. Then we can configure the default with `config._add('flask', dict(service='flask'))`

Then we can fetch via `config.flask.get_service()` which will do the globally configured service name if set, or else use the service name set on the integration config.

This means we'll automatically get an option for people to control the service name for every integration which follows the behavior we want.

Reply:
Ignoring for now, this expands on the scope of this PR.

Reply:
After some discussion with @brettlangdon we decided that adding `DD_<integration>_SERVICE[_NAME]` support is outside the scope of this PR as only a few integrations currently use the service name environment variable(s).

Also there are inconsistencies in the usage of either `config.service` or `config.service_name` that have to be addressed. Again, outside the scope of this PR.